### PR TITLE
P2P Ethermint cluster w/ mock da layer

### PR DESCRIPTION
**Pull request by jbowen93**
This PR makes several notable changes to the ephemeral cluster. 

- Creates a new `ethermint/` dir with contains static config files and keys for an ethermint node
- Updates the `dalc/` dir to contain an `optimint_server.toml` config file to work with the newer version of the DALC
- Creates a new `docker/mock-dalc-docker-compose.yml` file to start a mock DALC server using the optimint-mockserv binary and a `ethermint-mock-docker-compose.yml` file to use this mock DALC server. The `mock-ethermint.sh` script uses these files.
- Creates both `old-ethermint-docker-compose.yml` and `old-dalc-docker-compose.yml` files to enable running a full ethermint cluster prior to the breaking changes at current HEAD. The `old-minimal-ethermint.sh` script uses these files. 
- Modifies the `ethermint-docker-compose.yml` file to include 2 ethermint nodes with the first utilizing the new `ethermint/` dir's static config files and keys. This enables the second node to use the first node a `p2p.seed`. 

resolves: https://github.com/celestiaorg/ephemeral-cluster/issues/17

### CODE REVIEW

File: dalc/optimint_server.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[base]
+  laddr = "0.0.0.0:4200"
```
Comment (adlerjohn) [inline dalc/optimint_server.toml:2]: Indentation isn't needed for toml. I'd remove it just because I've literally never seen a toml file with indentation of non-multiline-fields before.

File: dalc/celestia-light/config.toml [hunk 2]
```diff
@@ -2,7 +2,7 @@
   Remote = false
   [Core.RemoteConfig]
     Protocol = "tcp"
-    RemoteAddr = "192.167.10.2:26657"
+    RemoteAddr = "192.167.10.0:26657"
```
Comment (adlerjohn) [inline dalc/celestia-light/config.toml:5]: Why this change?

File: docker/dalc-docker-compose.yml [hunk 3]
```diff
@@ -3,20 +3,26 @@ version: '3'
...
       - MY_VAL_ADDRESS=celes1dm7fp7uwrr5y68v48gg8jhd2cku7vt7f4l8w0s
+      - CELESTIA_PRIVATE_GENESIS=54A8B66D2BEF13850D67C8D474E196BD7485FE5A79989E31B17169371B0A9C96
     command: [
-            "dalc", "start",
-            "--home", "/root"
+            "celestia", "light",
+            "--node.store", "/dalc",
+            "start",
+            "--headers.trusted-peers",
+            "/ip4/192.167.1.0/tcp/2121/p2p/12D3KooWL8z3KARAYJcmExhDsGwKbjChKeGaJpFPENyADdxmEHzw",
```
Comment (adlerjohn) [inline docker/dalc-docker-compose.yml:16]: Why is this one `192.167.1.0`? Is there some list of which IPs do what?
```suggestion
            "/ip4/192.167.10.0/tcp/2121/p2p/12D3KooWL8z3KARAYJcmExhDsGwKbjChKeGaJpFPENyADdxmEHzw",
```

File: dalc/celestia-light/config.toml (hunk 2 above)
Comment (jbowen93) [reply dalc/celestia-light/config.toml:5]: This is the correct value, but because we currently spin up 3 core nodes that serve on 192.167.10.0 , 192.167.10.1, and 192.167.10.2 it worked before.

File: docker/dalc-docker-compose.yml (hunk 3 above)
Comment (jbowen93) [reply docker/dalc-docker-compose.yml:16]: There's no holistic list in one place. The `-docker-compose.yml` files have them
```
networks:
      localnet:
        ipv4_address: 192.167.3.0 
 ```
 the rough pattern is they all start with `192.167`. 
 - Core nodes(s)are on `192.167.10.x`
 - Bridge node(s) are on `192.167.1.x`
 - Light node(s) are on `192.167.2.x`
 - DALC is on `192.167.3.x`
 - Ethermint is on `192.167.4.x`

File: dalc/optimint_server.toml (hunk 1 above)
Comment (jbowen93) [reply dalc/optimint_server.toml]: Fixed.

**Review by adlerjohn — APPROVED** [review]
utACK
